libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -26,7 +26,7 @@ export function Dialog({
 }: DialogProps) {
   const contentClasses = classNames(
     // Positions the modal in the center of screen
-    'z-20 fixed w-full md:w-[520px] lg:w-[1000px] px-28 py-24 top-[50%] left-[50%] translate-x-[-50%] translate-y-[-50%]',
```

Comment:
question (blocking): i believe this was a requirement agreed by @sam-keen and @PeteWilliams that the modal sizes should be uniform and that particular size to accommodate the node switcher, because it looked pretty cramped under ~850px.
I've ended up changing this to 960px >[here](https://github.com/vegaprotocol/frontend-monorepo/pull/741/files#diff-1a450fe1bce39104bcd5b2c6b764f71945dbbd98f3f5b4bb7aae39138bda92acR29)< to prevent overflows, as it turns out our lg breakpoint os 960px.